.travis.yml

```diff
@@ -2,5 +2,5 @@ language: rust
 script:
   - cargo build --verbose
   - cargo test --verbose
-  - cargo build --verbose -p datomish-cli
```

Comment:
I realized that we didn't need to call `cargo build --verbose -p datomish-cli` in .travis.yml since a build error in datomish-cli will fail a build on datomish